services/web/server/src/simcore_service_webserver/studies_dispatcher/handlers_redirects.py

```diff
@@ -77,9 +77,8 @@ def decode_downloadlink(cls, v):
     @classmethod
     def from_request(cls, request: web.Request) -> "RedirectionQueryParams":
         try:
-            obj = cls(**dict(request.query))
+            obj = cls.parse_obj(dict(request.query))
```

Comment:
what about parse_orm ?

Reply:
``request.query`` returns a ``MultiDict`` and ``.parse_orm`` is of [ORM-like](https://pydantic-docs.helpmanual.io/usage/models/#orm-mode-aka-arbitrary-class-instances) instances, i.e. with dotted attributes 
for instance ``a.foo`` vs ``a["foo"]``. The former is ORM while for the latter one shall parse it with ``.parse_obj``